usaspending_api/awards/serializers.py

```diff
@@ -254,7 +254,6 @@ class Meta:
             "type_description",
             "category",
             "total_obligation",
-            "total_outlay",
```

Comment:
What is this doing?

Reply:
Removing the no-longer existing columns from the v1 API serializer